adblock_social_filters/adblock_social_list.txt

```diff
@@ -4004,8 +4009,7 @@ pcsupport.lenovo.com##.icon-l-ytb
 pct24.pl,pp24.info.pl,psz24.info##.socialtop, .socmed
 pdasite.pl,seniorka-z-plecakiem.pl,slodkieokruszki.pl###text-11
 pebx.pl##.bigFooterCol--col3.bigFooterCol
-pecetowicz.pl##.ipsGrid_span2
-pecetowicz.pl##.ipsList_reset > li.ipsBox.ipsWidget_vertical.ipsWidget:nth-of-type(3)
+pecetowicz.pl#?##secondaryFooter .ipsGrid_span3:nth-last-child(1) h4 + p+p:-abp-contains(Obserwuj)
```

Comment:
czemu p+p bez ozdobnych spacji

Reply:
merge or not to merge?

Reply:
Spacja wielkiej róznicy nie robi, więc merge :smile: